.travis.yml

```diff
@@ -1,12 +1,35 @@
+---
 language: node_js
 node_js:
```

Comment:
The node.js versions should probably be updated to include more recent ones.
Should the addon support 0.10?